benchmark/websocket/simple.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const common = require('../common.js');
+const crypto = require('crypto');
+const http = require('http');
+const { WebSocket } = require('undici');
+
+const GUID = '258EAFA5-E914-47DA-95CA-C5AB0DC85B11';
+
+const configs = {
+  size: [64, 16 * 1024, 128 * 1024, 1024 * 1024],
+  useBinary: ['true', 'false'],
+  roundtrips: [5000, 1000, 100, 1],
+};
+
+const bench = common.createBenchmark(main, configs);
+
+function createFrame(data, opcode) {
+  let infoLength = 2;
+  let payloadLength = data.length;
+
+  if (payloadLength >= 65536) {
+    infoLength += 8;
+    payloadLength = 127;
+  } else if (payloadLength > 125) {
+    infoLength += 2;
+    payloadLength = 126;
+  }
+
+  const info = Buffer.alloc(infoLength);
+
+  info[0] = opcode | 0x80;
+  info[1] = payloadLength;
+
+  if (payloadLength === 126) {
+    info.writeUInt16BE(data.length, 2);
+  } else if (payloadLength === 127) {
+    info[2] = info[3] = 0;
+    info.writeUIntBE(data.length, 4, 6);
+  }
+
+  return Buffer.concat([info, data]);
+}
+
+function main(conf) {
+  const frame = createFrame(Buffer.alloc(conf.size).fill('.'), 1);
```

Comment:
```suggestion
  const frame = createFrame(Buffer.alloc(conf.size).fill('.'), conf.useBinary ? 2 : 1);
```

Reply:
In order to avoid using Boolean type, I plan to modify it like this.
`const frame = createFrame(Buffer.alloc(conf.size).fill('.'), conf.useBinary === 'true');`